proto/src/determined/api/v1/api.proto

```diff
@@ -688,7 +689,7 @@ service Determined {
   // Control the job queue.
   rpc UpdateJobQueue(UpdateJobQueueRequest) returns (UpdateJobQueueResponse) {
     option (google.api.http) = {
-      post: "/api/v1/jobs/queue-ctl"
+      post: "/api/v1/resource-pools/queue/queue-ctl"
```

Comment:
🤔 could be a post on /rp/queue but it's not exactly creating a queue. 
maybe a patch would fit?  `patch /rp/queue`

Reply:
just a patch against the queue is looking better to me. thoughts?

Reply:
i had thought just posting a queue_action? patching the queue makes sense to me too (probably more than my dumb post idea)

Reply:
let's go with a patch now